hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/ECKeyOutputStream.java

```diff
@@ -587,8 +578,17 @@ public static void padBufferToLimit(ByteBuffer buf, int limit) {
     buf.position(limit);
   }
 
-  private boolean isPartialStripe() {
-    return currentBlockGroupLen % (numDataBlks * ecChunkSize) > 0;
+  private boolean isPartialStripe(int stripeSize) {
+    return stripeSize > 0 && stripeSize < numDataBlks * ecChunkSize;
+  }
+
+  private int getCurrentStripeSize() {
```

Comment:
Could you please rename this to getCurrentDataStripeSize?
Generally we refer stripe size as combination of data+parity. We we need only data.

Reply:
Sure, thanks.